packages/bot-web-ui/src/stores/self-exclusion-store.js

```diff
@@ -0,0 +1,66 @@
+import { observable, action, computed, runInAction } from 'mobx';
+
+export default class SelfExclusionStore {
+    constructor(root_store) {
+        this.root_store = root_store;
+        this.dbot = this.root_store.dbot;
+    }
+
+    @observable max_losses = '';
+    @observable set_max_losses = '';
+    @observable run_limit = '';
+    @observable is_restricted = false;
+
+    @computed
+    get initial_values() {
+        return {
+            set_max_losses: this.set_max_losses || '',
+            run_limit: this.run_limit,
+        };
+    }
+
+    @action.bound
+    setLimitations(key, value) {
+        this[key] = value;
+    }
+
+    @action.bound
+    checkClientRestriction() {
```

Comment:
This could be a `computed` value as the calculation/outcome is based on observables.
- https://mobx.js.org/refguide/computed-decorator.html

Reply:
> This could be a `computed` value as the calculation/outcome is based on observables.
> 
> * https://mobx.js.org/refguide/computed-decorator.html
Done

Reply:
The function is still the same, the `is_restricted` value could be `computed` value since it's a calculation based on `!client.is_eu && !client.is_virtual && (!this.max_losses || this.run_limit === -1)`.